backend/database/startup.py

```diff
@@ -12,6 +13,13 @@
 
 logger = logging.getLogger(__name__)
 
+redis_server = os.getenv("REDIS_HOST", "localhost")
```

Comment:
I'm wondering if we want to make this a single file that the others import

Reply:
Are you referring to a single "env" file? If that's the case, I've added an env.example file at the root that I use with docker. In my environment that file is called '.env' but they are being filtered out in .gitignore. I've added the env.example so that people can use it if they want.
In any case, I've set a default value on every 'getenv' so even if that file is not existing, there shouldn't be a problem

Reply:
^ This would be a great place to put all of these constants:
https://github.com/SaltieRL/DistributedReplays/blob/master/backend/server_constants.py

Then you can simply import constants in each of the places that they're used, so if the environment variable and/or default need to change, it changes in one spot

Reply:
I don't understand how I missed the server_constants file... I'll put them there.
As for the getenv with default values, I believe they don't do any harm. If we are going to fix them on the server_constants.py file, then there shouldn't be the need to check for them.
You tell me what do you prefer

Reply:
I think using the constants file for naming and defaults will be fine and we can leave the calls where they are

Reply:
Alright, I'll move them there

Reply:
While doing the changes I realized that those values are also needed by docker in the .env file.
Given the fact that they are infrastructure related, I am not sure if they should be added into the code. Or even if I add them into server_constants, I will still need them on .env (or use the default values as I've set them up).
What do you suggest?

Reply:
If they're defined in docker-compose, doesn't Docker pick them up?

Reply:
They are defaulted in docker compose but they are picked up from .env file.

Reply:
Anything defined in `.env` will override what you set in docker. If you want environment variables to be development-specific and not picked up by docker, you could defined a `.env.development` file. What's the exact issue you're running into?

Reply:
There's no issue. The question is about the migration of those definitions to the servers_constant file. If I do that, I will still need the . env file and ideally they should be defined in only one place

Reply:
You can't link the server constants in the .env file?
I am okay then if we import the .env file everywhere.

I just want to make sure they are only defined in one place.

Reply:
They could be added there. But they are going to be statically defined, not like python calculates them now

Reply:
In fact, I'm thinking that having the folders defined by python would prevent from moving the storage of the files or of the server, like to a bucket. How does production manage those now?

Reply:
files are already stored in buckets.
Nothing is stored locally in production

Reply:
ok, but you use a mounting point on the server for that bucket, right? You don't use URL.
When I separated Celery from Flask I had to create sort of a shared drive which is easy to do with docker. In Kubernetes, this is not as easy (or I am not finding it ez for now).
Anyway, I am lost now xD
What do you want me to do? I don't mind fixing/changing/cancelling :D

Reply:
For the bucket it is a URL
https://github.com/SaltieRL/DistributedReplays/blob/master/backend/utils/file_manager.py

Everything should go through here which knows how to read from the bucket.

I think we likely do not need to do anything

Reply:
Maybe not but I am struggling with Kubernetes to have a volume shared between different pods (Flask & Celery). Since in PRD Celery is replaced with a GCP Function then this is just an issue in dev. Unfortunately I ran over my subscription credit on Azure (6 days after the new month smh) where I was testing these things so I'll need to wait until next month.

In any case, and regarding this PR, I will cancel it because I found an issue with the proxy in package.json. I'll bring the discussion to discord